src/opensles/AudioInputStreamOpenSLES.cpp

```diff
@@ -57,24 +57,15 @@ AudioInputStreamOpenSLES::AudioInputStreamOpenSLES(const AudioStreamBuilder &bui
 AudioInputStreamOpenSLES::~AudioInputStreamOpenSLES() {
 }
 
-#define AUDIO_CHANNEL_COUNT_MAX         30u
-#define SL_ANDROID_UNKNOWN_CHANNELMASK  0
-
 int AudioInputStreamOpenSLES::chanCountToChanMask(int channelCount) {
```

Comment:
Why is this method in this *input stream* class when it refers to outputs? It feels like it should be a static/utility method elsewhere